linera-indexer/example/tests/test.rs

```diff
@@ -133,11 +134,15 @@ async fn test_end_to_end_operations_indexer(config: impl LineraNetConfig) {
         hash: None,
         chain_id: chain0,
     };
-    let last_block = request::<Block, _>(&req_client, "http://localhost:8080", variables)
-        .await
-        .unwrap()
-        .block
-        .unwrap_or_else(|| panic!("no block found"));
+    let last_block = request::<Block, _>(
+        &req_client,
+        &format!("http://localhost:{node_service_port}"),
+        variables,
+    )
+    .await
+    .unwrap()
+    .block
+    .unwrap_or_else(|| panic!("no block found"));
```

Comment:
```suggestion
    .expect("no block found");
```

Reply:
Doesn't clippy prefer `unwrap_or_else(|| ...)` because it's lazy?

Reply:
Yes, if the `foo` in `unwrap_or(foo())` does any work. But in this case it's just a `&'static str`.